spec/onchain.tex

```diff
@@ -414,19 +475,60 @@ \subsection{Close Transaction}\label{sec:close-tx}
 		\[
 			\eta' = \eta
 		\]
-		\item $\mathsf{Current}$: Closing snapshot refers to current state version $v$ and any UTxO to decommit need to be present in the closed state too.
+    \red{
+		  \item $\mathsf{UnusedInc}$: Closing snapshot refers to current state version $v$ and any UTxO to increment must be $\bot$ in the closed state.
+		  \[
+		  	\eta_\alpha' = \eta_\alpha = \eta_\Delta' = \bot
```

Comment:
What is $\eta_\alpha'$ here?

Note that $=$ is **not** an assignment here, but an equality. So if you use variables, they need to refer to something.


I don't think that we need the $\eta_\alpha'$ and $\eta_\omega'$ in any of these cases